internal/pkg/service/buffer/definition/source_base.go

```diff
@@ -0,0 +1,16 @@
+package definition
+
+import "github.com/keboola/keboola-as-code/internal/pkg/service/buffer/definition/key"
+
+type SourceType string
+
+type Source struct {
+	key.SourceKey
```

Comment:
`Source` entita, predtym sme to volali `Receiver`.

Je to pripravene, ze sa tam mozno neskor prida napr `KafkaSourceType`.